contributors/devel/bazel.md

```diff
@@ -1,43 +1,59 @@
 # Build with Bazel
 
-Building with bazel is currently experimental. Automanaged BUILD rules have the
+Building with Bazel is currently experimental. Automanaged `BUILD` rules have the
 tag "automanaged" and are maintained by
-[gazel](https://github.com/mikedanese/gazel). Instructions for installing bazel
+[gazel](https://github.com/mikedanese/gazel). Instructions for installing Bazel
 can be found [here](https://www.bazel.io/versions/master/docs/install.html).
 
-To build docker images for the components, run:
+Several `make` rules have been created for common operations:
 
-```
-$ bazel build //build/...
-```
+* `make bazel-build`: builds all binaries in tree
+* `make bazel-test`: runs all unit tests
+* `make bazel-release`: builds release tarballs, Docker images (for server
+  components), and Debian images
 
-To run many of the unit tests, run:
+You can also interact with Bazel directly; for example, to run all `kubectl` unit
+tests, run
 
-```
-$ bazel test //cmd/... //build/... //pkg/... //federation/... //plugin/...
+```console
+$ bazel test //pkg/kubectl/...
 ```
 
-To update automanaged build files, run:
+## Continuous Integration
 
-```
-$ ./hack/update-bazel.sh
-```
+The [Bazel CI job](http://k8s-testgrid.appspot.com/google-unit#bazel) runs
+`make bazel-build`, `make bazel-test`, and (transitively) `make bazel-release`.
+A similar job is run on all PRs.
 
-**NOTES**: `update-bazel.sh` only works if check out directory of Kubernetes is "$GOPATH/src/k8s.io/kubernetes".
+Many steps are cached, so the Bazel job usually executes fairly quickly.
 
-To update a single build file, run:
+## Known issues
 
-```
-$ # get gazel
-$ go get -u github.com/mikedanese/gazel
-$ # .e.g. ./pkg/kubectl/BUILD
-$ gazel -root="${YOUR_KUBE_ROOT_PATH}" ./pkg/kubectl
+[Cross-compilation is not currently supported](https://github.com/bazelbuild/rules_go/issues/70),
+so all binaries will be built for the host architecture running Bazel.
+Additionally, Go build tags are not supported. This means that builds on macOS may not work.
```

Comment:
This should be a separate paragraph/bullet point as it isn't a cross compilation issue.  This is a "broken on the mac" issue.  Building on the mac for the mac just craps out right now.